applications/nrf_desktop/nrf_desktop_description.rst

```diff
@@ -0,0 +1,1321 @@
+~~~~~~~~~~~~~~~~~~~~~~
+
+The nRF Desktop application defines the following common device identifiers:
+
+* Manufacturer (:ref:`CONFIG_DESKTOP_DEVICE_MANUFACTURER <config_desktop_app_options>`)
+* Vendor ID (:ref:`CONFIG_DESKTOP_DEVICE_VID <config_desktop_app_options>`)
+* Product name (:ref:`CONFIG_DESKTOP_DEVICE_PRODUCT <config_desktop_app_options>`)
+* Product ID (:ref:`CONFIG_DESKTOP_DEVICE_PID <config_desktop_app_options>`)
+
+These Kconfig options determine the default values of device identifiers used for:
+
+* :ref:`nrf_desktop_usb_state_identifiers`
+* BLE GATT Device Information Service (:kconfig:option:`CONFIG_BT_DIS`) that is required for :ref:`nrf_desktop_bluetooth_guide_peripheral`
+
+.. note::
+   Apart from the mentioned common device identifiers, the nRF Desktop application defines an application-specific string representing device generation (:ref:`CONFIG_DESKTOP_DEVICE_GENERATION <config_desktop_app_options>`).
+   The generation allows to distinguish configurations that use the same board and bootloader, but are not interoperable.
+   The value can be read through the :ref:`nrf_desktop_config_channel`.
+
+Debug configuration
+-------------------
+
+The nRF Desktop application introduces application-specific configuration options related to the debug configuration.
+These options are defined in the :file:`Kconfig.debug` file.
+
+The :ref:`CONFIG_DESKTOP_LOG <config_desktop_app_options>` Kconfig option enables support for logging in the nRF Desktop application.
+This option overlays Kconfig option defaults from the Logging subsystem to align them with the nRF Desktop requirements.
+The nRF Desktop configuration uses SEGGER J-Link RTT as the Logging subsystem backend.
+
+The :ref:`CONFIG_DESKTOP_SHELL <config_desktop_app_options>` Kconfig option enables support for CLI in the nRF Desktop application.
+This option overlays Kconfig option defaults from the Shell subsystem to align them with the nRF Desktop requirements.
+The nRF Desktop configuration uses SEGGER J-Link RTT as the Shell subsystem backend.
+If both shell and logging are enabled, logger uses shell as the logging backend.
+
+See the :file:`Kconfig.debug` file content for details.
+
+Default common configuration
+----------------------------
+
+The nRF Desktop application aligns the configuration with the nRF Desktop use case by overlaying Kconfig defaults and selecting or implying the required Kconfig options.
+Among others, the Kconfig :ref:`app_event_manager` and :ref:`lib_caf` options are selected to ensure that they are enabled.
+The :ref:`CONFIG_DESKTOP_SETTINGS_LOADER <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_POWER_MANAGER <config_desktop_app_options>` are implied to enable the :ref:`nrf_desktop_settings_loader` and :ref:`nrf_desktop_power_manager` modules, respectively.
+See the :file:`Kconfig.defaults` file for details related to default common configuration.
+
+.. _nrf_desktop_bluetooth_configuration:
+
+Bluetooth configuration
```

Comment:
Just a question - at the end of this section there's a redirect to https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-13938/nrf/applications/nrf_desktop/nrf_desktop_integration.html#bluetooth-in-nrf-desktop. Is there a reason from the user's perspective to keep the Bluetooth section this size in the main article? Would it make sense to e.g., shorten it significantly and just redirect the user to the more detailed article?

Reply:
Bluetooth configuration explains application-specific configuration options related to Bluetooth support and https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-13938/nrf/applications/nrf_desktop/nrf_desktop_integration.html#bluetooth-in-nrf-desktop is more about Bluetooth support in nRF desktop, where Bluetooth configuration can considered as subset. But, I think the current structure looks okay to me.

@MarekPieta : Could you share your thought on this?

Reply:
The `Bluetooth configuration` is a high-level overview of application-specific Kconfig options (I wanted to place it in configuration subsection to highlight it - some options are applied by default only by our application so it's worth to ensure that user is aware). The linked section describes Bluetooth setup in greater detail (since it's pretty long it was placed in a separate section).

Reply:
Thanks guys, makes sense, all clear:)